requirements/skinny-requirements.txt

```diff
@@ -16,3 +16,4 @@ sqlparse<1,>=0.4.0
 cachetools<6,>=5.0.0
 opentelemetry-api<3,>=1.9.0
 opentelemetry-sdk<3,>=1.9.0
+databricks-sdk<1,>=0.20.0
```

Comment:
Can we move this in the `databricks` extra? Only Databricks users need this.

Reply:
This might cause breaking change:

Previously databricks user can install mlflow in their laptop by `pip install mlflow` and then it can connect to databricks endpoint if tracking URI is set to `databricks`,

after applying this change "move this in the databricks extra", user has to install by `pip install mlflow[databricks]` otherwise user's existing code is broken.

and I think `databricks-sdk` is a very lightweight package, install it as the default dependency also makes user easier to migrate to our product, user doesn't need to install extra [databricks] .

Reply:
which packages does `databricks-sdk` depend on?

Reply:
databricks-sdk depends on `requests` and `google-auth` .

> Can we just show a helpful message if databricks-sdk is missing?

this sounds not ideal , it doesn't prevent the breaking change.